Otec/NewsRepository.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  NewsRepository.swift
+//  Otec
+//
+//  Created by El gera de la gente on 5/24/16.
+//  Copyright © 2016 Inaka. All rights reserved.
+//
+
+import Foundation
+import Jayme
+import IKEventSource
+class NewsRepository: CRUDRepository {
+    
+    typealias EntityType = News
+    let backend = NSURLSessionBackend()
+    let name = "news"
+    
+    func findAll(withEventSource eventSource: EventSource, newReceivedCompletion completion: News? -> ()) {
```

Comment:
I would rename this method because it does not match the expectations of a `findAll()`. It doesn't even return a `Future` so it works differently, regarding what you should expect from any repository's `findAll()`.